modules/sdpolicy-networking.adoc

```diff
@@ -57,7 +57,7 @@ Customers can determine their public static IP addresses by running a pod on the
 
 [source,terminal]
 ----
-$ oc run ip-lookup --image=busybox -i -t --restart=Never --rm -- /bin/sh -c "/bin/nslookup
+oc run ip-lookup --image=busybox -i -t --restart=Never --rm -- /bin/sh -c "/bin/nslookup -type=a myip.opendns.com resolver1.opendns.com | grep -E 'Address: [0-9.]+'"
```

Comment:
@cblecker thank you for submitting this pull request. The code block requires a `$` prompt in this case.

Reply:
@pneedle-rh Updated!

Reply:
Thank you!